<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeQQClientClass</class>
 <widget class="QWidget" name="WeQQClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WeQQClient</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>711</width>
     <height>531</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLineEdit" name="leName">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>180</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leServerPort">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>300</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leServerIp">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>260</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>180</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>260</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>服务器IP(默认127.0.0.1)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>300</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>服务器端口(默认9870)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConnect">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>340</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lePassword">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>220</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>220</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>201</width>
       <height>501</height>
      </rect>
     </property>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>161</width>
        <height>101</height>
       </rect>
      </property>
      <widget class="QPushButton" name="btnAddFriend">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>添加好友</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnJoinGroup">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>加入群组</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCreateGroup">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>10</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>创建群组</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leSearch">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>113</width>
         <height>23</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>输入ID/名</string>
       </property>
      </widget>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>161</width>
        <height>381</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>好友</string>
       </attribute>
       <widget class="QListWidget" name="listFriends">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>11</y>
          <width>141</width>
          <height>331</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>群组</string>
       </attribute>
       <widget class="QListWidget" name="listGroups">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>141</width>
          <height>331</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>481</width>
       <height>501</height>
      </rect>
     </property>
     <widget class="QLabel" name="labelChatTitle">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;未选择会话&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="browserChat">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>431</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="editMsg">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>431</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSend">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>470</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </widget>
    <zorder>widget_2</zorder>
    <zorder>widget</zorder>
   </widget>
   <widget class="QWidget" name="page_3"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="WeQQClient.qrc"/>
 </resources>
 <connections/>
</ui>
